src/payment/strategies/zip/zip-payment-strategy.ts

```diff
@@ -59,7 +62,10 @@ export default class ZipPaymentStrategy implements PaymentStrategy {
             throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
         }
 
-        return this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(payment.methodId))
+        return this._store.dispatch(this._remoteCheckoutActionCreator.initializePayment(payment.methodId, { useStoreCredit }))
```

Comment:
I assume you need to do this because the client token generation requires the order amount with store credit applied? Is it possible for you to just create the order first? Unless it's not really possible, you should always try to create the order first before forwarding order details to a hosted payment method. If you do that, you don't really need to do that extra call because the order amount will be final with the store credit applied.